launch/interaction/action_server.launch

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
There is no replacement launch file in [action_server](https://github.com/tue-robotics/action_server) yet